docs/core/linux-prerequisites.md

```diff
@@ -88,145 +105,184 @@ The installer bash script is used in automation scenarios and non-admin installa
     scl enable rh-dotnetcore11 bash
     ```
-4. Run the `dotnet --help` command to prove the installation succeeded.
+5. Run the `dotnet --help` command to prove the installation succeeded.
 
      ```bash
      dotnet --help
      ```
-> [!NOTE]
-> For Red Hat .NET channel access registration help, see [Chapter 1 of the .NET Core 1.1 Getting Started Guide](https://access.redhat.com/documentation/en/net-core/1.1/paged/getting-started-guide/) at Red Hat.
 
+For Red Hat .NET channel access registration help, see [Chapter 1 of the .NET Core 1.1 Getting Started Guide](https://access.redhat.com/documentation/en/net-core/1.1/paged/getting-started-guide/) at Red Hat.
+
+## Install .NET Core for Ubuntu 14.04, Ubuntu 16.04, Ubuntu 16.10 & Linux Mint 17, Linux Mint 18 (64 bit)
 
-## Install .NET Core for Ubuntu 14.04, 16.04, 16.10 & Linux Mint 17, 18 (64 bit)
+1. Verify that all previous .NET Core preview versions have been removed.
 
 > [!Warning]
-> Before you start, remove any previous versions of .NET Core from your system.
+> Before you start, remove any **previous preview** versions of .NET Core from your system.
 
 ### Add the dotnet apt-get feed
 
-1. Set up the desired version host package feed.
+2. Set up the desired version host package feed.
 
    **Ubuntu 14.04 / Linux Mint 17**
+
     ```bash
     sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list' 
     sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
     sudo apt-get update
     ```
+
     **Ubuntu 16.04 / Linux Mint 18**
+
     ```bash
     sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ xenial main" > /etc/apt/sources.list.d/dotnetdev.list' 
     sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
     sudo apt-get update
     ```
+
     **Ubuntu 16.10**
+
     ```bash
     sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ yakkety main" > /etc/apt/sources.list.d/dotnetdev.list' 
     sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
     sudo apt-get update
     ```
-2. Install .NET Core.
+
+3. Install .NET Core. 
+
 # [.NET Core 2.x](#tab/netcore2x)
 
-After host package feed setup, install .NET Core 2.0 on Ubuntu or Linux Mint:
+Install .NET Core 2.0 on Ubuntu or Linux Mint:
+
 ```bash
 sudo apt-get install dotnet-sdk-2.0.0
 ```
+
 # [.NET Core 1.x](#tab/netcore1x)
 
-After host package feed setup, install .NET Core 1.x on Ubuntu or Linux Mint:
+Install .NET Core 1.x on Ubuntu or Linux Mint:
+
 ```bash
 sudo apt-get install dotnet-dev-1.0.4
 ```
+
 ---
-3. Run the `dotnet --help` command to prove the installation succeeded.
 
- ```bash
-     dotnet --help
- ```
-## Install .NET Core for Debian 8 or 9 (64 bit).
+4. Run the `dotnet --help` command to prove the installation succeeded.
```

Comment:
I think it would be even better to do dotnet --version. So you can confirm you have the version you want in case you installed multiple.